Sources/ProjectSpec/Scheme.swift

```diff
@@ -620,6 +623,7 @@ extension Scheme.Test: JSONObjectConvertible {
         customLLDBInit = jsonDictionary.json(atKeyPath: "customLLDBInit")
         captureScreenshotsAutomatically = jsonDictionary.json(atKeyPath: "captureScreenshotsAutomatically") ?? Scheme.Test.captureScreenshotsAutomaticallyDefault
         deleteScreenshotsWhenEachTestSucceeds = jsonDictionary.json(atKeyPath: "deleteScreenshotsWhenEachTestSucceeds") ?? Scheme.Test.deleteScreenshotsWhenEachTestSucceedsDefault
+        macroExpansion = jsonDictionary.json(atKeyPath: "macroExpansion")
```

Comment:
Sorry, just missing the encoding in `toJSONValue` below as well

Reply:
done https://github.com/yonaskolb/XcodeGen/pull/1468/commits/01953369c4d4bba397ac893d65c072b65b3287da